app/models/subjects.rb

```diff
@@ -0,0 +1,51 @@
+module Subjects
+  SUBJECT_OPTIONS = [
```

Comment:
Maybe have these subjects in YAML format? Kind of what we have done with boroughs, cities, counties and regions